static/cfg-view.js

```diff
@@ -127,7 +128,7 @@ define(function (require) {
         this.setTitle();
     }
 
-    Cfg.prototype.onCompileResult = function (id, compiler, result) {
+    Cfg.prototype.onCompileResult = function (id, compiler, result, binaryFilter) {
```

Comment:
As discussed on slack, a "filtersChange" which passes a compilerId and the effective filters is probably a better idea than modifying onCompileResult
Also, Jared's suggestion of only sending the diff would probably be a good idea but I think it falls out of scope for the PR